docs/source/notebooks/power_analysis.ipynb

```diff
@@ -0,0 +1,1412 @@
+{
```

Comment:
<p>Can you use the following syntax for the note, and it will render as a nice admonition box:</p><p> </p><p>:::{note}</p><p>MY TEXT HERE</p><p>:::</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/CausalPy/pull/277/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
Done, should look better now? I think is not render well for my Jupyter

Reply:
Looks good to me
![Screenshot 2024-01-05 at 16 21 00](https://github.com/pymc-labs/CausalPy/assets/6765047/eb50fb37-1a37-4af2-9734-7388f7f3696c)